doc/getting-started/rt_industry.rst

```diff
@@ -0,0 +1,390 @@
+   |                     |                        | set up Service VM or Native OS           |                                                                                                |
+   +---------------------+------------------------+------------------------------------------+------------------------------------------------------------------------------------------------+
+
+Install build tools and dependencies
+====================================
+On the Clear Linux build machine, install the necessary tools in order to build ACRN
+hypervisor, Service VM and Preempt-RT kernels::
+
+  $ sudo swupd bundle-add os-clr-on-clr os-core-dev python3-basic
+  $ echo "export PATH=\$HOME/.local/bin:\$PATH" >> ~/.bashrc && source ~/.bashrc
+  $ pip3 install --user kconfiglib
+
+How to set up ACRN Hypervisor for industry scenario
+***************************************************
+In this section, we'll follow the :ref:`ACRN quick setup guide <quick-setup-guide>`
+to set up the Service VM. Then we'll set up the ACRN hypervisor by enabling
+industry scenario.
+
+Using the pre-installed industry ACRN hypervisor
+================================================
+The industry hypervisor image is installed in ``/usr/lib/acrn//usr/lib/acrn/`` directory
+once the Service VM is set up. You may simply follow these steps to use
+``acrn.kbl-nuc-i7.industry.efi`` instead of the original SDC hypervisor::
+
+  $ sudo mount /dev/sda1 /mnt
+  $ sudo mv /mnt/EFI/acrn/acrn.efi /mnt/EFI/acrn/acrn.efi.bak
+  $ sudo cp /usr/lib/acrn/acrn.kbl-nuc-i7.industry.efi /mnt/EFI/acrn/acrn.efi
+  $ sync && umount /mnt
+  $ sudo reboot
+
+Follow the next section if you want to generate an industry hypervisor image from the source code.
+
+Using the industry ACRN hypervisor built from source code
+=========================================================
+#. Follow these commands to generate the ``acrn.efi`` with industry scenario::
+
+   $ cd ~
+   $ git clone http://github.com/projectacrn/acrn-hypervisor.git
+   $ cd acrn-hypervisor
+   $ git checkout v1.2
+   $ make BOARD=kbl-nuc-i7 SCENARIO=industry FIRMWARE=uefi hypervisor
+
+#. Login Service VM, use industry ``acrn.efi`` generated above::
+
+   $ sudo mount /dev/sda1 /mnt
+   $ sudo mv /mnt/EFI/acrn/acrn.efi /mnt/EFI/acrn/acrn.efi.bak
+   $ sudo scp <build machine>:~/acrn-hypervisor/hypervisor/build/acrn.efi  /mnt/EFI/acrn/
+   $ sync && umount /mnt
+   $ sudo reboot
+
+Update Preempt-RT kernel
+************************
+In this section, we will learn to update the Preempt-RT kernel by two methods:
+
+- Rebuild from acrn-kernel branch tree.
+- Install Preempt-RT kernel from Clear Linux bundles.
+
+You could try either of them to get the Preempt-RT kernel and then copy it onto NVMe disk.
+
+Rebuild Preempt-RT kernel from acrn-kernel branch tree
+======================================================
+#. Build the Preempt-RT kernel on your build machine::
+
+   $ cd ~/acrn-kernel
```

Comment:
This assumes the repo has been cloned before but it's missing from the instructions, can you add it?

Reply:
Will add this instruction regarding to the "Build kernel" into "Build ACRN from Source", so remove   it here. 😄